workflow/schemas/rules.schema.yaml

```diff
@@ -1130,7 +1130,137 @@ properties:
           phasing:
             type: string
             description: file with phasing information
-  
+
+
+  savana_pb_to:
+    type: object
+    description: input and output parameters for savana_pb_to
+    properties:
+      input:
+        type: object
+        description: list of inputs
+        properties:
+          bam:
+            type: string
+            description: mapped and haplotagged BAM file
+          bai:
+            type: string
+            description: BAM index file
+          ref:
+            type: string
+            description: reference genome file in FASTA format
+          fai:
+            type: string
+            description: reference genome index file
+
```

Comment:
Add output files here for all four rules. Also, skip the extra newline before output:.